src/scenes/Root/Root.tsx

```diff
@@ -31,6 +32,7 @@ export const Root = () => {
       <Route path="/" element={<Home />} />
       <Route path="/organizations/*" element={<Organizations />} />
       <Route path="/projects/*" element={<Projects />} />
+      <Route path="/projects/:projectId" element={<ProjectOverview />} />
```

Comment:
This should be declared in [`Projects`](https://github.com/SeedCompany/cord-field/blob/a11aa3df971e406f2f95e256171492981bc3aa13/src/scenes/Projects/Projects.tsx#L10) instead

Reply:
Moved into Projects component